MaxText/configs/a3/llama_2_7b/16vm.sh

```diff
@@ -30,5 +30,5 @@ export XLA_FLAGS="--xla_dump_to=$OUTPUT_PATH/$RUN_NAME/HLO_dumps/
 --xla_disable_hlo_passes=rematerialization"
 
 # 16 nodes
-python MaxText/$EXECUTABLE MaxText/configs/models/gpu/llama2_7b.yml run_name=$RUN_NAME \
+python3 -m MaxText.$EXECUTABLE MaxText/configs/models/gpu/llama2_7b.yml run_name=$RUN_NAME \
```

Comment:
Moving this comment from [PR 1478](https://github.com/AI-Hypercomputer/maxtext/pull/1478) to this main PR

Thank you @SamuelMarks for these contributions. Just for my understanding, why is this change needed? Might be missing where you mentioned it somewhere.

I understand this is related to running `python -m pytest` and `pytest`. But how is this related to the issue with running those?